Source/VMware.vSphereDSC/Tests/Integration/VMHostVss/VMHostVssShaping.Integration.Tests.ps1

```diff
@@ -0,0 +1,197 @@
+#>
+
+param(
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Name,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Server,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $User,
+
+    [Parameter(Mandatory = $true)]
+    [string]
+    $Password
+)
+
+$script:moduleName = 'VMware.vSphereDSC'
+$script:dscResourceName = 'VMHostVssShaping'
+$script:dscDependResourceName = 'VMHostVss'
+$script:moduleFolderPath = (Get-Module -Name $script:moduleName -ListAvailable).ModuleBase
+$script:integrationTestsFolderPath = Join-Path (Join-Path $moduleFolderPath 'Tests') 'Integration'
+$script:configurationFile = "$script:integrationTestsFolderPath\Configurations\$($script:dscResourceName)\$($script:dscResourceName)_Config.ps1"
+
+$script:configWithModifyVssShaping = "$($script:dscResourceName)_Modify_Config"
+$script:configWithRemoveVssShaping = "$($script:dscResourceName)_Remove_Config"
+
+$script:VssName = 'VSSDSC'
+$script:AverageBandwidth = 100000
+$script:BurstSize = 100000
+$script:Enabled = $false
+$script:PeakBandwidth = 100000
+$script:Present = 'Present'
+$script:Absent = 'Absent'
+
+$script:resourceWithModifyVssShaping = @{
+    Name = $Name
+    Server = $Server
+    Ensure = $script:Present
+    VssName = $script:VssName
+    AverageBandwidth = $script:AverageBandwidth + 1
+    BurstSize = $script:BurstSize + 1
+    Enabled = -not $script:Enabled
+    PeakBandwidth = $script:PeakBandwidth + 1
+}
+$script:resourceWithRemoveVssShaping = @{
+    Name = $Name
+    Server = $Server
+    Ensure = $script:Absent
+    VssName = $script:VssName
+    AverageBandwidth = $script:AverageBandwidth
+    BurstSize = $script:BurstSize
+    Enabled = $script:Enabled
+    PeakBandwidth = $script:PeakBandwidth
+}
+
+. $script:configurationFile -Name $Name -Server $Server -User $User -Password $Password
+
+$script:mofFileWithModifyVssShaping = "$script:integrationTestsFolderPath\$($script:configWithModifyVssShaping)\"
+$script:mofFileWithRemoveVssShaping = "$script:integrationTestsFolderPath\$($script:configWithRemoveVssShaping)\"
+
+try {
+    Describe "$($script:dscResourceName)_Integration" {
+        Context "When using configuration $($script:configWithModifyVssShaping)" {
+            BeforeEach {
+                # Arrange
+                $startDscConfigurationParameters = @{
+                    Path = $script:mofFileWithModifyVssShaping
+                    ComputerName = 'localhost'
+                    Wait = $true
+                    Force = $true
+                }
+
+                # Act
+                Start-DscConfiguration @startDscConfigurationParameters
+            }
+
+            It 'Should compile and apply the MOF without throwing' {
+                # Assert
+                $startDscConfigurationParameters = @{
+                    Path = $script:mofFileWithModifyVssShaping
+                    ComputerName = 'localhost'
+                    Wait = $true
+                    Force = $true
+                }
```

Comment:
1. Add empty line here and also move the # Assert comment here.

```
 # Assert
 { Start-DscConfiguration @startDscConfigurationParameters } | Should -Not -Throw
```

The same comments for the other tests.

Reply:
Changed

Reply:
Do the same thing for the second Configuration tests.

Reply:
Changed